src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFMerger.java

```diff
@@ -17,27 +22,42 @@
 @Service
 public class PDFMerger {
 
-    public File merge(Bundle bundle, Map<BundleDocument, File> documents) throws IOException {
-        StatefulPDFMerger statefulPDFMerger = new StatefulPDFMerger(documents, bundle);
+    public File merge(Bundle bundle, Map<BundleDocument, File> documents, String caseData) throws IOException, DocumentTaskProcessingException {
+        StatefulPDFMerger statefulPDFMerger = new StatefulPDFMerger(documents, bundle, caseData);
 
         return statefulPDFMerger.merge();
     }
 
     private class StatefulPDFMerger {
         private final PDFMergerUtility merger = new PDFMergerUtility();
         private final PDDocument document = new PDDocument();
+        private final ObjectMapper mapper = new ObjectMapper();
         private TableOfContents tableOfContents;
         private final Map<BundleDocument, File> documents;
         private final Bundle bundle;
+        private final JsonNode caseData;
         private static final String BACK_TO_TOP = "Back to top";
         private int currentPageNumber = 0;
 
-        public StatefulPDFMerger(Map<BundleDocument, File> documents, Bundle bundle) {
+        @Autowired
+        private TemplateRenditionClient templateRenditionClient;
+
+        public StatefulPDFMerger(Map<BundleDocument, File> documents, Bundle bundle, String caseData) throws IOException {
             this.documents = documents;
             this.bundle = bundle;
+            this.caseData = mapper.readTree(caseData);
         }
 
-        public File merge() throws IOException {
+        public File merge() throws IOException, DocumentTaskProcessingException {
+            if (bundle.getCoverpageTemplate() != null
```

Comment:
Can we use this StringUtils.isNotEmpty(). Better may be this StringUtils.isNotBlank()